gfop/get_food_counts.py

```diff
@@ -70,12 +71,18 @@ def get_file_food_counts(gnps_network, sample_types, all_groups, some_groups,
     filenames = (df_selected['UniqueFileSources'].str.split('|')
                  .explode())
     # Select food hierarchy levels.
-    sample_types = sample_types[f'sample_type_group{level}'] # remove this line to do file-level
+    if level in range(1, 7):
+        sample_types = sample_types[f'sample_type_group{level}']
```

Comment:
Even shorter could be `sample_types = sample_types[f'sample_type_group{level}' if level > 0 else 'sample_name']`